src/database/user.ts

```diff
@@ -130,4 +130,28 @@ export default class UserData {
       throw error;
     }
   }
+
+  static async getUserById(userId: number): Promise<User> {
+    try {
+      const response = await db
+        .db("sep6._user")
+        .select("*")
+        .where("user_id", userId)
+        .then((rows) => {
+          if (rows === undefined || rows.length == 0) {
+            throw Error("User could not be found");
+          }
+          return {
+            userId: rows[0].user_id,
+            username: rows[0].username,
+            email: rows[0].email,
+            password: rows[0].password,
+          };
```

Comment:
Can't you just return rows[0]?

Reply:
Maybe. This is according to the other methods that are already there. Is this a problem, though? If it works.

Reply:
It will not be an object anymore, and it would return more items than required as our sql rn has profile pic etc...

Reply:
and in this part he maps everything so that it would be compatible with our user object.